Resources/Views/Home/home.leaf

```diff
@@ -36,16 +36,22 @@
   <div class="container">
     #extend("Home/_about-us")
     <br/>
+    #extend("Home/_highlights")
+    <br/>
     #if(ticketsEnabled):
         #extend("Home/_order-tickets")
     #endif
     #if(count(speakers) > 0):
-        <br/>
-        #extend("Home/_speakers")
+        #if(conferenceEnded):
+            <br/>
+            #extend("Home/_speakers")
+        #endif
```

Comment:
Is the intention to have this inverted? If the conference is ended, don't show the speakers/schedule. This currently reads wrong vs the PR description.

Reply:
My mistake, pushing this up now 😅 good spot!